### PR TITLE
adds a new fallout based tacti-cool kit and ports laser rifle from bad deathclaw

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
was listening to some fallout music and i thought to add a fallout based kit as we already have some fallout items altoids made for me months ago.
currently contains
a aer 9 laser rifle which does 40 damage per hit but has a 5 second shot cooldown
a t45b helmet
a t45b armor
a stimpack
image

#### Changelog

:cl:  
rscadd: Added a new gun and a new kit
imageadd: ported some bad deathclaw icons
/:cl:

### CODE REVIEW

**Comment by nesisknesis** [comment]
pictures, icon bot is kill

**Comment by boodaliboo** [comment]
> pictures, icon bot is kill

do my pictures even work im doing this as a web edit

**Review by nesisknesis — CHANGES_REQUESTED** [review]
images

**Comment by boodaliboo** [comment]
oh theres aconflit one sec

**Comment by boodaliboo** [comment]
should work now

**Comment by boodaliboo** [comment]
WHAT IS EVEN PREVENTING IT

**Comment by Mr-Space** [comment]
travis upsetti
code spaghetti

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -120,8 +120,8 @@
 			new /obj/item/card/emag(src) // 6 tc
 
 /obj/item/storage/box/syndicate/bundle_B/PopulateContents()
-	switch (pickweight(list( "bond" = 2, "neo=1", "ninja" = 1, "darklord" = 1, "white_whale_holy_grail" = 2, "mad_scientist" = 2, "bee" = 2, "mr_freeze" = 2)))
-		if("bond")
+	switch (pickweight(list( "bond" = 2,"neo=1", "brother" = 1, "ninja" = 1, "darklord" = 1, "white_whale_holy_grail" = 2, "mad_scientist" = 2, "bee" = 2, "mr_freeze" = 2)))
+    	if("bond")
```
Comment (missatessatessy) [inline code/game/objects/items/storage/uplink_kits.dm]: There's something up with the indentation here which is what is throwing travis off to start with.

Comment (missatessatessy) [reply code/game/objects/items/storage/uplink_kits.dm]: Still not right

Comment (missatessatessy) [reply code/game/objects/items/storage/uplink_kits.dm]: Try copying the the three lines from /obj to if, from the original code and pasting it in again (adding the brother)?

Comment (missatessatessy) [reply code/game/objects/items/storage/uplink_kits.dm]: Also just noticed neo is wrong, should say "neo" = 1 but it says "neo=1"

**Comment by alexkar598** [comment]
That gun is stupid OP

**Comment by boodaliboo** [comment]
> That gun is stupid OP

with a cooldown of 4 seconds and with this being a rng only item i feel like its fairly balanced

**Comment by nesisknesis** [comment]
syndicate dont get lasers gun theyre too dumb

**Comment by boodaliboo** [comment]
> syndicate dont get lasers gun theyre too dumb

they get lots of wacky tech

**Comment by alexkar598** [comment]
Conflict, not interested and noone seems willing to merge this or even look at it

**Pull request closed without merging**